lib/reversetunnel/agent.go

```diff
@@ -361,6 +357,7 @@ func (a *Agent) startHeartbeat(conn ssh.Conn) {
 			errC <- err
 			return
 		}
+		hb.SendRequest("ping", false, nil)
```

Comment:
but that's not what going on at least from what I read - in this particular case it simply sends ping twice